GPIOOneWire/CPP/ReadMe.md

```diff
@@ -0,0 +1,253 @@
+# GPIO OneWire DHT11/22 reader
+
+These are the available versions of this Windows 10 IoT Enterprise sample:
+
+* [C++](./CPP/ReadMe.md)
+
+| Minimum SDK Version | 19041 |
+|---------------------|-------|
+| Minimum OS Version  | 19041 |
+
+This sample has been updated to use the `GpioChangeReader` API which is
+new as of build 19041. You must install SDK version 19041 or later to build
+this sample.
+
+The sample has been temporarily modified to use two pins instead of one to
+drive the DHT22. `GpioChangeReader` requires interrupts to be
+enabled. `GpioPin.SetDriveMode()` currently takes
+much longer when interrupts are enabled than when they are
+disabled. If you try to change a pin from output to input when interrupts
+are enabled, you will miss the response from the DHT22.
+As a workaround, two pins are used to interface with the DHT22.
+One pin is configured as an output and is used to assert the data line LOW
+to request a sample. The other pin is configured as input and is used to
+capture changes on the data line.
+
+Your breadboard should be wired as follows. You can use either a MOSFET or

```

Comment:
We probably want to have part numbers here, people are going to ask what BJT or MOSFET they should use.

Reply:
Agreed - please add @v-abdrahman

Reply:
Added the name of BJT and MOSFET in the readmes